### DISCUSSION TITLE
Usage

### DISCUSSION THREAD

**Discussion by jeffepok** [EA31337/EA31337-Libre#35] [Q&A]
Can you please expand on the usage? How do I get it working on my MT5 terminal

**Comment by kenorb** [comment]
Please have a look at the [Installation page](https://github.com/EA31337/EA31337/wiki/Installation).
It should be straight-forward. You have to download the binary from the Releases page into platform's _Experts/_ directory, then drag and drop the EX file into the active chart, or run the backtest using _Strategy Tester_.

**Reply by jeffepok** [reply]
Thanks
